### ISSUE TITLE
On Yellow enabling UART support for HM_MOD_RPI_PCB in config.txt removes internal Zigbee device /dev/ttyAMA1

### ISSUE THREAD

**Issue by traumtheater** [home-assistant/operating-system#2408]
### Describe the issue you are experiencing

When trying to use the Momematic RF hat HM_MOD_RPI_PCB in conjunction with the integrated Zigbee module of Yellow it seems that there is a conflict in the UART connection.
When enabling the configuration in config.txt as specified in the documentation of the RaspberryMatic Home-Assistant Add-On ([https://github.com/jens-maus/RaspberryMatic/wiki/Installation-HomeAssistant]), the Homematic RF hat works but the internal Zigbee device is unavailable. The log states:
```
Error setting up entry Yellow for zha
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/serial/serialposix.py", line 322, in open
    self.fd = os.open(self.portstr, os.O_RDWR | os.O_NOCTTY | os.O_NONBLOCK)
FileNotFoundError: [Errno 2] No such file or directory: '/dev/ttyAMA1'
```
I learned that a Z-Wave hat which uses the same pinning as the HM_MOD_RPI_PCB works directly out of the box without changing any dtparams.

### What operating system image do you use?

yellow (Home Assistant Yellow)

### What version of Home Assistant Operating System is installed?

9.5

### Did you upgrade the Operating System.

No

### Steps to reproduce the issue

1. Attach HM_MOD_RPI_PCB to J11 header of Yellow
2. Edit /mnt/boot/config.txt and uncomment the following three entries:
```
dtparam=i2c_arm=on
dtoverlay=miniuart-bt
dtoverlay=rpi-rf-mod
```
3. Fully reboot the system

Now internal Zigbee is not found anymore and Zigbee is therefore unavailable but HM_MOD_RPI_PCB is usable in RaspberryMatic and the Homematic integration.

### Anything in the Supervisor logs that might be useful for us?

```txt
23-03-14 15:12:00 INFO (MainThread) [supervisor.plugins.multicast] Starting Multicast plugin
23-03-14 15:12:01 INFO (SyncWorker_0) [supervisor.docker.multicast] Starting Multicast ghcr.io/home-assistant/aarch64-hassio-multicast with version 2022.02.0 - Host
23-03-14 15:12:01 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
23-03-14 15:12:01 INFO (SyncWorker_2) [supervisor.docker.interface] Attaching to ghcr.io/home-assistant/yellow-homeassistant with version 2023.3.4
23-03-14 15:12:01 INFO (MainThread) [supervisor.homeassistant.secrets] Loaded 63 Home Assistant secrets
23-03-14 15:12:02 INFO (MainThread) [supervisor.os.manager] Detect Home Assistant Operating System 9.5 / BootSlot A
23-03-14 15:12:03 WARNING (SyncWorker_0) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of DeskPi Pro Active Cooling
23-03-14 15:12:04 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/de838cd8 repository
23-03-14 15:12:04 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/5c53de3b repository
23-03-14 15:12:04 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/2ad4c73a repository
23-03-14 15:12:04 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/a0d7b954 repository
23-03-14 15:12:04 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/45df7312 repository
23-03-14 15:12:04 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/git/1f3d020e repository
23-03-14 15:12:04 INFO (MainThread) [supervisor.store.git] Loading add-on /data/addons/core repository
23-03-14 15:12:09 WARNING (SyncWorker_1) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of DeskPi Pro Active Cooling
23-03-14 15:12:09 INFO (MainThread) [supervisor.store] Loading add-ons from store: 90 all - 90 new - 0 remove
23-03-14 15:12:09 INFO (MainThread) [supervisor.addons] Found 9 installed add-ons
23-03-14 15:12:09 INFO (SyncWorker_7) [supervisor.docker.interface] Attaching to 1f3d020e/aarch64-addon-hassos_ssh_configurator_addon with version 0.9.1a
23-03-14 15:12:09 INFO (SyncWorker_3) [supervisor.docker.interface] Attaching to homeassistant/aarch64-addon-mosquitto with version 6.1.3
23-03-14 15:12:09 INFO (SyncWorker_4) [supervisor.docker.interface] Attaching to ghcr.io/hassio-addons/phpmyadmin/aarch64 with version 0.8.4
23-03-14 15:12:09 INFO (SyncWorker_2) [supervisor.docker.interface] Attaching to homeassistant/aarch64-addon-mariadb with version 2.5.2
23-03-14 15:12:09 INFO (SyncWorker_5) [supervisor.docker.interface] Attaching to ghcr.io/jens-maus/raspberrymatic with version 3.67.10.20230225
23-03-14 15:12:09 INFO (SyncWorker_0) [supervisor.docker.interface] Attaching to homeassistant/aarch64-addon-ssh with version 9.6.1
23-03-14 15:12:09 INFO (SyncWorker_6) [supervisor.docker.interface] Attaching to homeassistant/aarch64-addon-configurator with version 5.5.0
23-03-14 15:12:09 INFO (SyncWorker_1) [supervisor.docker.interface] Attaching to ghcr.io/esphome/esphome-hassio with version 2023.2.4
23-03-14 15:12:09 INFO (SyncWorker_7) [supervisor.docker.interface] Attaching to ghcr.io/hassio-addons/tailscale/aarch64 with version 0.7.0
23-03-14 15:12:09 INFO (MainThread) [supervisor.backups.manager] Found 23 backup files
23-03-14 15:12:09 INFO (MainThread) [supervisor.discovery] Loaded 2 messages
23-03-14 15:12:09 INFO (MainThread) [supervisor.ingress] Loaded 7 ingress sessions
23-03-14 15:12:09 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.SETUP
23-03-14 15:12:09 INFO (MainThread) [supervisor.resolution.check] System checks complete
23-03-14 15:12:09 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
23-03-14 15:12:09 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-03-14 15:12:09 INFO (MainThread) [supervisor.jobs] 'ResolutionFixup.run_autofix' blocked from execution, system is not running - CoreState.SETUP
23-03-14 15:12:09 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.SETUP
23-03-14 15:12:09 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-03-14 15:12:09 INFO (MainThread) [__main__] Running Supervisor
23-03-14 15:12:09 INFO (MainThread) [supervisor.os.manager] Rauc: A - marked slot kernel.0 as good
23-03-14 15:12:09 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.INITIALIZE' starting 0 add-ons
23-03-14 15:12:09 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.SYSTEM' starting 2 add-ons
23-03-14 15:12:11 INFO (SyncWorker_1) [supervisor.docker.addon] Starting Docker add-on homeassistant/aarch64-addon-mosquitto with version 6.1.3
23-03-14 15:12:12 INFO (SyncWorker_6) [supervisor.docker.addon] Starting Docker add-on homeassistant/aarch64-addon-mariadb with version 2.5.2
23-03-14 15:12:16 INFO (MainThread) [supervisor.services.modules.mqtt] Set core_mosquitto as service provider for mqtt
23-03-14 15:12:17 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.SERVICES' starting 3 add-ons
23-03-14 15:12:18 INFO (SyncWorker_2) [supervisor.docker.addon] Starting Docker add-on homeassistant/aarch64-addon-ssh with version 9.6.1
23-03-14 15:12:19 INFO (MainThread) [supervisor.services.modules.mysql] Set core_mariadb as service provider for MySQL
23-03-14 15:12:19 INFO (SyncWorker_1) [supervisor.docker.addon] Starting Docker add-on ghcr.io/jens-maus/raspberrymatic with version 3.67.10.20230225
23-03-14 15:12:19 INFO (SyncWorker_6) [supervisor.docker.addon] Starting Docker add-on ghcr.io/hassio-addons/tailscale/aarch64 with version 0.7.0
23-03-14 15:12:24 INFO (MainThread) [supervisor.api.middleware.security] /network/interface/default/info access from a0d7b954_tailscale
23-03-14 15:12:24 INFO (MainThread) [supervisor.core] Start Home Assistant Core
23-03-14 15:12:24 INFO (SyncWorker_3) [supervisor.docker.interface] Starting homeassistant
23-03-14 15:12:25 INFO (MainThread) [supervisor.homeassistant.core] Wait until Home Assistant is ready
23-03-14 15:12:34 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/eq3loop - None
23-03-14 15:12:34 INFO (MainThread) [supervisor.docker.addon] Added cgroup permissions 'c 235:0 rwm' for device /dev/eq3loop to addon_de838cd8_raspberrymatic
23-03-14 15:12:35 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.STARTUP
23-03-14 15:12:35 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-03-14 15:12:35 INFO (MainThread) [supervisor.homeassistant.api] Updated Home Assistant API token
23-03-14 15:12:35 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
23-03-14 15:12:35 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/adamoutler/Addons repository
23-03-14 15:12:35 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/hassio-addons/repository repository
23-03-14 15:12:35 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/adamoutler/HassOSConfigurator repository
23-03-14 15:12:35 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/jens-maus/RaspberryMatic repository
23-03-14 15:12:35 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/zigbee2mqtt/hassio-zigbee2mqtt repository
23-03-14 15:12:36 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/esphome/home-assistant-addon repository
23-03-14 15:12:36 INFO (MainThread) [supervisor.store.git] Update add-on https://github.com/home-assistant/addons repository
23-03-14 15:12:37 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/mmd_hmip - None
23-03-14 15:12:37 INFO (MainThread) [supervisor.hardware.monitor] Detecting HardwareAction.ADD hardware /dev/mmd_bidcos - None
23-03-14 15:12:37 INFO (MainThread) [supervisor.docker.addon] Added cgroup permissions 'c 235:1 rwm' for device /dev/mmd_hmip to addon_de838cd8_raspberrymatic
23-03-14 15:12:37 INFO (MainThread) [supervisor.docker.addon] Added cgroup permissions 'c 235:2 rwm' for device /dev/mmd_bidcos to addon_de838cd8_raspberrymatic
23-03-14 15:12:38 WARNING (SyncWorker_0) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of DeskPi Pro Active Cooling
23-03-14 15:12:39 WARNING (SyncWorker_6) [supervisor.addons.validate] Add-on have full device access, and selective device access in the configuration. Please report this to the maintainer of DeskPi Pro Active Cooling
23-03-14 15:12:39 INFO (MainThread) [supervisor.store] Loading add-ons from store: 90 all - 0 new - 0 remove
23-03-14 15:12:39 INFO (MainThread) [supervisor.store] Loading add-ons from store: 90 all - 0 new - 0 remove
23-03-14 15:13:27 INFO (MainThread) [supervisor.homeassistant.core] Detect a running Home Assistant instance
23-03-14 15:13:27 INFO (MainThread) [supervisor.addons] Phase 'AddonStartup.APPLICATION' starting 2 add-ons
23-03-14 15:13:28 INFO (SyncWorker_2) [supervisor.docker.addon] Starting Docker add-on ghcr.io/esphome/esphome-hassio with version 2023.2.4
23-03-14 15:13:29 INFO (SyncWorker_1) [supervisor.docker.addon] Starting Docker add-on homeassistant/aarch64-addon-configurator with version 5.5.0
23-03-14 15:13:34 INFO (MainThread) [supervisor.misc.tasks] All core tasks are scheduled
23-03-14 15:13:34 INFO (MainThread) [supervisor.core] Supervisor is up and running
23-03-14 15:13:34 INFO (MainThread) [supervisor.host.info] Updating local host information
23-03-14 15:13:34 INFO (MainThread) [supervisor.updater] Fetching update data from https://version.home-assistant.io/stable.json
23-03-14 15:13:34 INFO (MainThread) [supervisor.resolution.check] Starting system checks with state CoreState.RUNNING
23-03-14 15:13:34 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.NO_CURRENT_BACKUP/ContextType.SYSTEM
23-03-14 15:13:34 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.FREE_SPACE/ContextType.SYSTEM
23-03-14 15:13:34 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.PWNED/ContextType.ADDON
23-03-14 15:13:35 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.IPV4_CONNECTION_PROBLEM/ContextType.SYSTEM
23-03-14 15:13:35 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.TRUST/ContextType.SUPERVISOR
23-03-14 15:13:35 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_IPV6_ERROR/ContextType.DNS_SERVER
23-03-14 15:13:35 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.DNS_SERVER_FAILED/ContextType.DNS_SERVER
23-03-14 15:13:35 INFO (MainThread) [supervisor.resolution.checks.base] Run check for IssueType.SECURITY/ContextType.CORE
23-03-14 15:13:35 INFO (MainThread) [supervisor.resolution.check] System checks complete
23-03-14 15:13:35 INFO (MainThread) [supervisor.resolution.evaluate] Starting system evaluation with state CoreState.RUNNING
23-03-14 15:13:36 INFO (MainThread) [supervisor.host.services] Updating service information
23-03-14 15:13:36 INFO (MainThread) [supervisor.host.network] Updating local network information
23-03-14 15:13:36 INFO (MainThread) [supervisor.host.sound] Updating PulseAudio information
23-03-14 15:13:36 INFO (MainThread) [supervisor.host.manager] Host information reload completed
23-03-14 15:13:36 INFO (MainThread) [supervisor.resolution.evaluate] System evaluation complete
23-03-14 15:13:36 INFO (MainThread) [supervisor.resolution.fixup] Starting system autofix at state CoreState.RUNNING
23-03-14 15:13:36 INFO (MainThread) [supervisor.resolution.fixup] System autofix complete
```

### Anything in the Host logs that might be useful for us?

```txt
Mar 14 14:12:34 homeassistant kernel: audit: type=1334 audit(1678803154.528:325): prog-id=0 op=UNLOAD
Mar 14 14:12:34 homeassistant kernel: audit: type=1334 audit(1678803154.532:326): prog-id=67 op=LOAD
Mar 14 14:12:34 homeassistant kernel: audit: type=1300 audit(1678803154.532:326): arch=c00000b7 syscall=280 success=yes exit=12 a0=5 a1=4000196280 a2=78 a3=0 items=0 ppid=118 pid=5097 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="runc" exe="/usr/bin/runc" subj=unconfined key=(null)
Mar 14 14:12:34 homeassistant kernel: audit: type=1327 audit(1678803154.532:326): proctitle=72756E63002D2D726F6F74002F7661722F72756E2F646F636B65722F72756E74696D652D72756E632F6D6F62792F00757064617465002D2D7265736F7572636573002D00613266666339306639333431323063313263356563336163626238356630643762373963666430383361633538653536653635653131653663646462
Mar 14 14:12:34 homeassistant kernel: audit: type=1334 audit(1678803154.532:327): prog-id=68 op=LOAD
Mar 14 14:12:34 homeassistant kernel: audit: type=1300 audit(1678803154.532:327): arch=c00000b7 syscall=280 success=yes exit=14 a0=5 a1=4000196010 a2=78 a3=0 items=0 ppid=118 pid=5097 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="runc" exe="/usr/bin/runc" subj=unconfined key=(null)
Mar 14 14:12:34 homeassistant kernel: audit: type=1327 audit(1678803154.532:327): proctitle=72756E63002D2D726F6F74002F7661722F72756E2F646F636B65722F72756E74696D652D72756E632F6D6F62792F00757064617465002D2D7265736F7572636573002D00613266666339306639333431323063313263356563336163626238356630643762373963666430383361633538653536653635653131653663646462
Mar 14 14:12:34 homeassistant kernel: audit: type=1334 audit(1678803154.532:328): prog-id=0 op=UNLOAD
Mar 14 14:12:34 homeassistant kernel: audit: type=1334 audit(1678803154.532:329): prog-id=0 op=UNLOAD
Mar 14 14:12:34 homeassistant os-agent[118]: INFO: 2023/03/14 14:12:34 cgroup.go:66: Successfully called runc for 'a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447', output time="2023-03-14T14:12:34Z" level=warning msg="could not find device group for '99/166' in /proc/devices -- temporarily ignoring rule: {99 166 -1 rwm true}"
Mar 14 14:12:34 homeassistant os-agent[118]: time="2023-03-14T14:12:34Z" level=info msg="found more than one filter (2) attached to a cgroup -- removing extra filters!"
Mar 14 14:12:34 homeassistant os-agent[118]: time="2023-03-14T14:12:34Z" level=info msg="removing old filter 0 from cgroup" id=58 name= run_count=0 runtime=0s tag=4e4c7c5c1d856132 type=CGroupDevice
Mar 14 14:12:34 homeassistant os-agent[118]: time="2023-03-14T14:12:34Z" level=info msg="removing old filter 1 from cgroup" id=66 name= run_count=0 runtime=0s tag=a9735e97c1c45e48 type=CGroupDevice
Mar 14 14:12:34 homeassistant os-agent[118]: INFO: 2023/03/14 14:12:34 cgroup.go:69: Permission 'a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447', granted for Container 'c 235:0 rwm' via runc
Mar 14 14:12:35 homeassistant kernel: eq3loop: created slave mmd_hmip
Mar 14 14:12:35 homeassistant kernel: eq3loop: created slave mmd_bidcos
Mar 14 14:12:35 homeassistant systemd[1]: Bluetooth service was skipped because of a failed condition check (ConditionPathIsDirectory=/sys/class/bluetooth).
Mar 14 14:12:35 homeassistant systemd[1]: Bluetooth service was skipped because of a failed condition check (ConditionPathIsDirectory=/sys/class/bluetooth).
Mar 14 14:12:37 homeassistant os-agent[118]: INFO: 2023/03/14 14:12:37 cgroup.go:66: Successfully called runc for 'a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447', output time="2023-03-14T14:12:37Z" level=warning msg="could not find device group for '99/166' in /proc/devices -- temporarily ignoring rule: {99 166 -1 rwm true}"
Mar 14 14:12:37 homeassistant os-agent[118]: time="2023-03-14T14:12:37Z" level=info msg="found more than one filter (2) attached to a cgroup -- removing extra filters!"
Mar 14 14:12:37 homeassistant os-agent[118]: time="2023-03-14T14:12:37Z" level=info msg="removing old filter 0 from cgroup" id=69 name= run_count=0 runtime=0s tag=fa7cf28ee43556b7 type=CGroupDevice
Mar 14 14:12:37 homeassistant os-agent[118]: time="2023-03-14T14:12:37Z" level=info msg="removing old filter 1 from cgroup" id=70 name= run_count=0 runtime=0s tag=7bc0b02272839fcd type=CGroupDevice
Mar 14 14:12:37 homeassistant os-agent[118]: INFO: 2023/03/14 14:12:37 cgroup.go:69: Permission 'a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447', granted for Container 'c 235:1 rwm' via runc
Mar 14 14:12:37 homeassistant os-agent[118]: INFO: 2023/03/14 14:12:37 cgroup.go:66: Successfully called runc for 'a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447', output time="2023-03-14T14:12:37Z" level=warning msg="could not find device group for '99/166' in /proc/devices -- temporarily ignoring rule: {99 166 -1 rwm true}"
Mar 14 14:12:37 homeassistant os-agent[118]: time="2023-03-14T14:12:37Z" level=info msg="found more than one filter (2) attached to a cgroup -- removing extra filters!"
Mar 14 14:12:37 homeassistant os-agent[118]: time="2023-03-14T14:12:37Z" level=info msg="removing old filter 0 from cgroup" id=73 name= run_count=0 runtime=0s tag=88605d1cad057d8c type=CGroupDevice
Mar 14 14:12:37 homeassistant os-agent[118]: time="2023-03-14T14:12:37Z" level=info msg="removing old filter 1 from cgroup" id=74 name= run_count=0 runtime=0s tag=b13ad56aff7bc522 type=CGroupDevice
Mar 14 14:12:37 homeassistant os-agent[118]: INFO: 2023/03/14 14:12:37 cgroup.go:69: Permission 'a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447', granted for Container 'c 235:2 rwm' via runc
Mar 14 14:12:38 homeassistant kernel: eq3loop: eq3loop_open_slave() mmd_bidcos
Mar 14 14:12:38 homeassistant kernel: eq3loop: eq3loop_close_slave() mmd_bidcos
Mar 14 14:12:38 homeassistant kernel: eq3loop: eq3loop_open_slave() mmd_hmip
Mar 14 14:12:38 homeassistant kernel: eq3loop: eq3loop_close_slave() mmd_hmip
Mar 14 14:12:38 homeassistant kernel: eq3loop: eq3loop_open_slave() mmd_bidcos
Mar 14 14:12:53 homeassistant kernel: eq3loop: eq3loop_open_slave() mmd_hmip
Mar 14 14:12:53 homeassistant kernel: eq3loop: eq3loop_close_slave() mmd_hmip
Mar 14 14:12:53 homeassistant kernel: eq3loop: eq3loop_open_slave() mmd_hmip
Mar 14 14:12:53 homeassistant kernel: eq3loop: eq3loop_close_slave() mmd_hmip
Mar 14 14:12:53 homeassistant kernel: eq3loop: eq3loop_open_slave() mmd_hmip
Mar 14 14:12:53 homeassistant kernel: eq3loop: eq3loop_close_slave() mmd_hmip
Mar 14 14:12:53 homeassistant kernel: eq3loop: eq3loop_open_slave() mmd_hmip
Mar 14 14:13:27 homeassistant systemd[1]: var-lib-docker-overlay2-ba878a5f9d381ff3ab959be74bd5d329744f7ee2965a40dcbc509a3e59ad656c\x2dinit-merged.mount: Deactivated successfully.
Mar 14 14:13:27 homeassistant systemd[1]: mnt-data-docker-overlay2-ba878a5f9d381ff3ab959be74bd5d329744f7ee2965a40dcbc509a3e59ad656c\x2dinit-merged.mount: Deactivated successfully.
Mar 14 14:13:27 homeassistant dockerd[417]: time="2023-03-14T14:13:27.929509767Z" level=info msg="loading plugin \"io.containerd.event.v1.publisher\"..." runtime=io.containerd.runc.v2 type=io.containerd.event.v1
Mar 14 14:13:27 homeassistant dockerd[417]: time="2023-03-14T14:13:27.945535933Z" level=info msg="loading plugin \"io.containerd.internal.v1.shutdown\"..." runtime=io.containerd.runc.v2 type=io.containerd.internal.v1
Mar 14 14:13:27 homeassistant dockerd[417]: time="2023-03-14T14:13:27.945599449Z" level=info msg="loading plugin \"io.containerd.ttrpc.v1.task\"..." runtime=io.containerd.runc.v2 type=io.containerd.ttrpc.v1
Mar 14 14:13:27 homeassistant dockerd[417]: time="2023-03-14T14:13:27.946331551Z" level=info msg="starting signal loop" namespace=moby path=/run/docker/containerd/daemon/io.containerd.runtime.v2.task/moby/d8c08183be32f0130bf194d16b3823d066857e010e38a274a3a777ea41bc3f2f pid=6006 runtime=io.containerd.runc.v2
Mar 14 14:13:28 homeassistant systemd[1]: Started libcontainer container d8c08183be32f0130bf194d16b3823d066857e010e38a274a3a777ea41bc3f2f.
Mar 14 14:13:28 homeassistant kernel: kauditd_printk_skb: 32 callbacks suppressed
Mar 14 14:13:28 homeassistant kernel: audit: type=1334 audit(1678803208.058:348): prog-id=78 op=LOAD
Mar 14 14:13:28 homeassistant kernel: audit: type=1334 audit(1678803208.062:349): prog-id=79 op=LOAD
Mar 14 14:13:28 homeassistant kernel: audit: type=1300 audit(1678803208.062:349): arch=c00000b7 syscall=280 success=yes exit=16 a0=5 a1=4000143940 a2=78 a3=0 items=0 ppid=6006 pid=6015 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="runc" exe="/usr/bin/runc" subj=unconfined key=(null)
Mar 14 14:13:28 homeassistant kernel: audit: type=1327 audit(1678803208.062:349): proctitle=72756E63002D2D726F6F74002F7661722F72756E2F646F636B65722F72756E74696D652D72756E632F6D6F6279002D2D6C6F67002F7661722F72756E2F646F636B65722F636F6E7461696E6572642F6461656D6F6E2F696F2E636F6E7461696E6572642E72756E74696D652E76322E7461736B2F6D6F62792F64386330383138
Mar 14 14:13:28 homeassistant kernel: audit: type=1334 audit(1678803208.062:350): prog-id=80 op=LOAD
Mar 14 14:13:28 homeassistant kernel: audit: type=1300 audit(1678803208.062:350): arch=c00000b7 syscall=280 success=yes exit=18 a0=5 a1=40001436d0 a2=78 a3=0 items=0 ppid=6006 pid=6015 auid=4294967295 uid=0 gid=0 euid=0 suid=0 fsuid=0 egid=0 sgid=0 fsgid=0 tty=(none) ses=4294967295 comm="runc" exe="/usr/bin/runc" subj=unconfined key=(null)
Mar 14 14:13:28 homeassistant kernel: audit: type=1327 audit(1678803208.062:350): proctitle=72756E63002D2D726F6F74002F7661722F72756E2F646F636B65722F72756E74696D652D72756E632F6D6F6279002D2D6C6F67002F7661722F72756E2F646F636B65722F636F6E7461696E6572642F6461656D6F6E2F696F2E636F6E7461696E6572642E72756E74696D652E76322E7461736B2F6D6F62792F64386330383138
Mar 14 14:13:28 homeassistant kernel: audit: type=1334 audit(1678803208.062:351): prog-id=0 op=UNLOAD
Mar 14 14:13:28 homeassistant kernel: audit: type=1334 audit(1678803208.062:352): prog-id=0 op=UNLOAD
Mar 14 14:13:28 homeassistant kernel: audit: type=1334 audit(1678803208.062:353): prog-id=81 op=LOAD
Mar 14 14:13:28 homeassistant kernel: hassio: port 10(veth0f2944d) entered blocking state
Mar 14 14:13:28 homeassistant kernel: hassio: port 10(veth0f2944d) entered disabled state
Mar 14 14:13:28 homeassistant kernel: device veth0f2944d entered promiscuous mode
Mar 14 14:13:28 homeassistant systemd-udevd[6056]: Using default interface naming scheme 'v250'.
Mar 14 14:13:28 homeassistant NetworkManager[342]: <info>  [1678803208.3885] manager: (veth75c56e5): new Veth device (/org/freedesktop/NetworkManager/Devices/25)
Mar 14 14:13:28 homeassistant systemd-udevd[6057]: Using default interface naming scheme 'v250'.
Mar 14 14:13:28 homeassistant NetworkManager[342]: <info>  [1678803208.3974] manager: (veth0f2944d): new Veth device (/org/freedesktop/NetworkManager/Devices/26)
Mar 14 14:13:28 homeassistant dockerd[417]: time="2023-03-14T14:13:28.541324509Z" level=info msg="loading plugin \"io.containerd.event.v1.publisher\"..." runtime=io.containerd.runc.v2 type=io.containerd.event.v1
Mar 14 14:13:28 homeassistant dockerd[417]: time="2023-03-14T14:13:28.541979040Z" level=info msg="loading plugin \"io.containerd.internal.v1.shutdown\"..." runtime=io.containerd.runc.v2 type=io.containerd.internal.v1
Mar 14 14:13:28 homeassistant dockerd[417]: time="2023-03-14T14:13:28.542230382Z" level=info msg="loading plugin \"io.containerd.ttrpc.v1.task\"..." runtime=io.containerd.runc.v2 type=io.containerd.ttrpc.v1
Mar 14 14:13:28 homeassistant dockerd[417]: time="2023-03-14T14:13:28.543346156Z" level=info msg="starting signal loop" namespace=moby path=/run/docker/containerd/daemon/io.containerd.runtime.v2.task/moby/cd823ad67914f8257a9513de26ca1597a0d3f995e32b437e474cabeb731699cf pid=6094 runtime=io.containerd.runc.v2
Mar 14 14:13:28 homeassistant systemd[1]: Started libcontainer container cd823ad67914f8257a9513de26ca1597a0d3f995e32b437e474cabeb731699cf.
Mar 14 14:13:29 homeassistant kernel: eth0: renamed from veth75c56e5
Mar 14 14:13:29 homeassistant kernel: IPv6: ADDRCONF(NETDEV_CHANGE): veth0f2944d: link becomes ready
Mar 14 14:13:29 homeassistant kernel: hassio: port 10(veth0f2944d) entered blocking state
Mar 14 14:13:29 homeassistant kernel: hassio: port 10(veth0f2944d) entered forwarding state
Mar 14 14:13:29 homeassistant NetworkManager[342]: <info>  [1678803209.7789] device (veth0f2944d): carrier: link connected
Mar 14 14:13:34 homeassistant kernel: kauditd_printk_skb: 41 callbacks suppressed
Mar 14 14:13:34 homeassistant kernel: audit: type=1334 audit(1678803214.937:369): prog-id=86 op=LOAD
Mar 14 14:13:34 homeassistant kernel: audit: type=1334 audit(1678803214.941:370): prog-id=87 op=LOAD
Mar 14 14:13:34 homeassistant kernel: audit: type=1334 audit(1678803214.941:371): prog-id=88 op=LOAD
Mar 14 14:13:34 homeassistant systemd[1]: Starting Hostname Service...
Mar 14 14:13:35 homeassistant systemd[1]: Started Hostname Service.
Mar 14 14:13:35 homeassistant kernel: audit: type=1334 audit(1678803215.525:372): prog-id=89 op=LOAD
Mar 14 14:13:35 homeassistant kernel: audit: type=1334 audit(1678803215.525:373): prog-id=90 op=LOAD
Mar 14 14:13:35 homeassistant kernel: audit: type=1334 audit(1678803215.529:374): prog-id=91 op=LOAD
Mar 14 14:13:35 homeassistant systemd[1]: Starting Time & Date Service...
Mar 14 14:13:36 homeassistant systemd[1]: Started Time & Date Service.
Mar 14 14:13:50 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447-runc.wguJnv.mount: Deactivated successfully.
Mar 14 14:14:05 homeassistant systemd[1]: systemd-hostnamed.service: Deactivated successfully.
Mar 14 14:14:05 homeassistant kernel: audit: type=1334 audit(1678803245.637:375): prog-id=0 op=UNLOAD
Mar 14 14:14:05 homeassistant kernel: audit: type=1334 audit(1678803245.637:376): prog-id=0 op=UNLOAD
Mar 14 14:14:05 homeassistant kernel: audit: type=1334 audit(1678803245.637:377): prog-id=0 op=UNLOAD
Mar 14 14:14:06 homeassistant systemd[1]: systemd-timedated.service: Deactivated successfully.
Mar 14 14:14:06 homeassistant kernel: audit: type=1334 audit(1678803246.093:378): prog-id=0 op=UNLOAD
Mar 14 14:14:06 homeassistant kernel: audit: type=1334 audit(1678803246.093:379): prog-id=0 op=UNLOAD
Mar 14 14:14:06 homeassistant kernel: audit: type=1334 audit(1678803246.093:380): prog-id=0 op=UNLOAD
Mar 14 14:14:20 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447-runc.r6M3MK.mount: Deactivated successfully.
Mar 14 14:15:21 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447-runc.wyFwyW.mount: Deactivated successfully.
Mar 14 14:15:51 homeassistant systemd[1]: run-docker-runtime\x2drunc-moby-a2ffc90f934120c12c5ec3acbb85f0d7b79cfd083ac58e56e65e11e6cddbb447-runc.BsYPCe.mount: Deactivated successfully.
Mar 14 14:17:18 homeassistant kernel: audit: type=1334 audit(1678803438.217:381): prog-id=92 op=LOAD
Mar 14 14:17:18 homeassistant systemd[1]: Started Journal Gateway Service.
Mar 14 14:17:18 homeassistant systemd-journal-gatewayd[7132]: microhttpd: MHD_OPTION_EXTERNAL_LOGGER is not the first option specified for the daemon. Some messages may be printed by the standard MHD logger.
```

## System Information

version | core-2023.3.4
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.10.10
os_name | Linux
os_version | 5.15.84-v8
arch | aarch64
timezone | Europe/Berlin
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4982
Installed Version | 1.31.0
Stage | running
Available Repositories | 1248
Downloaded Repositories | 2

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | false
-- | --
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 9.5
-- | --
update_channel | stable
supervisor_version | supervisor-2023.03.1
agent_version | 1.4.1
docker_version | 20.10.22
disk_total | 457.7 GB
disk_used | 8.3 GB
healthy | true
supported | true
board | yellow
supervisor_api | ok
version_api | ok
installed_addons | ESPHome (2023.2.4), Terminal & SSH (9.6.1), Mosquitto broker (6.1.3), MariaDB (2.5.2), phpMyAdmin (0.8.4), File editor (5.5.0), RaspberryMatic CCU (3.67.10.20230225), Tailscale (0.7.0), HassOS SSH port 22222 Configurator (0.9.1a)

</details>

<details><summary>Dashboards</summary>

dashboards | 6
-- | --
resources | 1
views | 10
mode | storage

</details>

<details><summary>Recorder</summary>

error | failed to load: unknown
-- | --

</details>

**Comment by agners** [comment]
@jens-maus have you recently used/tested `rpi-rf-mod` on Yellow?

**Comment by jens-maus** [comment]
No I haven't actually tested that because I never thought someone would be so keen in using yellow with zigbee and a homematic module in parallel simply because of too high probability of RF interferences.

The issue could easily be the case because the generic `rpi-rf-mod.dts` will make sure to reserve the GPIO-based uart for the GPIO only. See:

https://github.com/jens-maus/RaspberryMatic/blob/57a7174bd55b4c180aefba23d8ba7b4735c0798a/buildroot-external/package/rpi-rf-mod/dts/rpi-rf-mod.dts#L75-L86

So there isn't a specific yellow dts file available yet which will make sure that the other uart will be untouched and working. Or might this be a side effect of using the `miniuart-bt` dtoverlay in parallel?

**Comment by traumtheater** [comment]
If I recall it correctly, not using miniuart-bt but the other options results in both Interfaces not working, neither internal Zigbee nor the Homematic hat.

**Comment by Mar1usW3** [comment]
i´m curious if this can be fixed or do i need an external zigbee stick?

**Comment by agners** [comment]
The overlay is not correct for Yellow, and probably breaks Audio as well.

I'll remove the HomeMatic overlay support for Yellow to avoid issues with the on-board Zigbee.

**Comment by agners** [comment]
The overlay won't be part of OS 10, which essentially resolves this issue.

I guess it would be nice to have a device tree overlay which works for `HM_MOD_RPI_PCB`.

> No I haven't actually tested that because I never thought someone would be so keen in using yellow with zigbee and a homematic module in parallel simply because of too high probability of RF interferences.

@jens-maus from what I understand Homeatic is in the 863MHz band, Zigbee is on the 2.4GHz band so these really shouldn't interfere with each other. From a general EMI perspective, we did proper CE certifications and are below the limits both variants, the PoE and Standard variant. And the Yellow Standard variant is even a lot below any limits.

That said, conducted noise from the power supplies etc, can always affect radio performance. We did design the power supplies with radio performance in mind, and at least the Zigbee performance seemed unaffected (as in we reached performance levels Silicon Labs has in their reference implementations).

I'd definitely say it is worth a try :smile:

**Comment by Chris2834** [comment]
> No I haven't actually tested that because I never thought someone would be so keen in using yellow with zigbee and a homematic module in parallel simply because of too high probability of RF interferences.
> 
> The issue could easily be the case because the generic `rpi-rf-mod.dts` will make sure to reserve the GPIO-based uart for the GPIO only. See:
> 
> https://github.com/jens-maus/RaspberryMatic/blob/57a7174bd55b4c180aefba23d8ba7b4735c0798a/buildroot-external/package/rpi-rf-mod/dts/rpi-rf-mod.dts#L75-L86
> 
> So there isn't a specific yellow dts file available yet which will make sure that the other uart will be untouched and working. Or might this be a side effect of using the `miniuart-bt` dtoverlay in parallel?

@jens-maus : I'm using #yellow and HM_MOD_RPI_PCB by USB connection  succesfull, because I could not get it running with J11 on #Yellow. 
Did not see this issue in time, but can read in your .dts code, that different GPIO's are blocked by standard .dts (and removed for yellow now) than used in #Yellow J11.
Yellow J11 is using GPIO 14/15  as TX/RX port. (See https://yellow.home-assistant.io/hardware/yellow-v1.2-standard/)

If you can give me a hint how to add a correct .dts file, I might check it.

**Comment by traumtheater** [comment]
> @jens-maus : I'm using #yellow and HM_MOD_RPI_PCB by USB connection succesfull, because I could not get it running with J11 on #Yellow. Did not see this issue in time, but can read in your .dts code, that different GPIO's are blocked by standard .dts (and removed for yellow now) than used in #Yellow J11. Yellow J11 is using GPIO 14/15 as TX/RX port. (See https://yellow.home-assistant.io/hardware/yellow-v1.2-standard/)
> If you can give me a hint how to add a correct .dts file, I might check it.

I ordered a HB-RF-USB-2 hoping that I can use the module via USB. Sadly also via this USB adapter my HM-MOD-RPI-PCB is not detected. How did you get it working via USB?

**Comment by Chris2834** [comment]
> I ordered a HB-RF-USB-2 hoping that I can use the module via USB. Sadly also via this USB adapter my HM-MOD-RPI-PCB is not detected. How did you get it working via USB?

Yep; I'm using the HB-RF-USB-2 without any HW /SW mod.

But I don't like the "external" wiring and additional box needed. 
Would still prefer the  J11 connection - potentially with add on antenna on the back side - to get a clean setup.

**Comment by traumtheater** [comment]
Ok this is a bummer. I really do not know why it is not working on my side. I also do not know how to debug the issue.
Is there a difference which USB port to connect? Or is the jumper setting on the PC important? I see there is a UART/USB setting for example...

**Comment by smoki3** [comment]
So does this mean we actually dont have a solution to use HM_MOD_RPI_PCB on the yellow GPIO Pins?

**Comment by agners** [comment]
Right now we don't have a tested/verified/working device tree overlay for Yellow, correct. I also don't have the necessary hardware at hand.

**Comment by smoki3** [comment]
Do you see a chance to have a short term work around? before we have another solution. (dont want to spent another 30 Euro for the USB variant)

Could it work if we remove all the RPI-RF-MOD stuff from the dts overlay? Because the MOD-RPI-PCB just need the GPIO14 and GPIO15.  Or do we need to use a different uart (for example uart1?)

**Comment by jens-maus** [comment]
> Do you see a chance to have a short term work around? before we have another solution. (dont want to spent another 30 Euro for the USB variant)

You mean, I should invest my time for you to save actually 30 EUR?!? ;-)

> Could it work if we remove all the RPI-RF-MOD stuff from the dts overlay? Because the MOD-RPI-PCB just need the GPIO14 and GPIO15. Or do we need to use a different uart (for example uart1?)

This is really IMHO not worth the trouble. Using a HB-RF-USB-2 together with a RPI-RF-MOD or using a HmIP-RFUSB is anyway the better approach than trying to get the obsolete HM-MOD-RPI-PCB running....

**Comment by Chris2834** [comment]
I can offer to spent my time on implementation and testing, if I get to know where to modify. I have a second yellow and RF available to check.

I did see the Yellow as best solution to convert my old rPi based Raspberrymatic to more options and reliable hardware with HomeAssistant Yellow - in using the GPIO with the RPI-RF-MOD. I would like to continue that road to go to eliminate as much wiring as possible.

**Comment by smoki3** [comment]
> You mean, I should invest my time for you to save actually 30 EUR?!? ;-)

I would rather spent you a coffee instead of buying new hardware and have a cable solution attached to the Yellow. I bought the yellow because this could be a all in one solution if the GPIOs will work. I think it will be very clean box will everythin included. 

> This is really IMHO not worth the trouble. Using a HB-RF-USB-2 together with a RPI-RF-MOD or using a HmIP-RFUSB is anyway the better approach than trying to get the obsolete HM-MOD-RPI-PCB running...

The HB-RF-USB-2 is very big, you need an extra box + cabels in the living room... Even with the HmIP-RFUSB you still have this ugly stick attached to the yellow. 


So the HM-MOD-RPI-PCB looks like the perfect accesory for the yellow. I fully agree with Chis2834. If we have a hint what to test I will do it.

**Comment by traumtheater** [comment]
As I have spend the money on the HB-RF-USB-2 and it is still not working I would like to offer my time as well for testing. Even if I somehow get the USB solution to work, I would also prefer a more integrated and clean setup. That is why I wanted to replace my Raspberry 1 with Raspberrymatic in the first place.

@jens-maus Thank you for the amazing support with Raspberrymatic!

**Comment by jens-maus** [comment]
> So the HM-MOD-RPI-PCB looks like the perfect accesory for the yellow. I fully agree with Chis2834. If we have a hint what to test I will do it.

Even if the HM-MOD-RPI-PCB looks like a neat solution on a first thought, it actually **is** obsolete/outdated, etc. No newer firmware updates exist for this homematic rf module it and will never get any further firmware updates released by the vendor (eQ3). Currently, the RPI-RF-MOD and HmIP-RFUSB homematic RF modules are the only ones I would recommend these days since they come with the latest 4.4.x firmwares themselves which provide superior homematic/homematicIP support in some areas.

Therefore, I would still suggest to better rely on a RPI-RF-MOD in combination with a HB-RF-USB-2 adapter or use a HmIP-RFUSB for homematic communication and refrain from using the old HM-MOD-RPI-PCB even thought it looks attractive to use it on the GPIO bus.

**Comment by Chris2834** [comment]
I hear all the good arguments, but I have still 3 pieces of HM-MOD-RPI-PCB running.
Just want to change to #yellow and my use case is ok with the original FW content of HM-MOD-RPI-PCB.

**Comment by jens-maus** [comment]
Let me please allow to re-open this ticket/issue as I am currently working on a HM-MOD-RPI-PCB specific device tree overlay for the Yellow which hopefully will not break Zigbee support.

**Comment by traumtheater** [comment]
A quick update from my experiments:
I can happily announce that the issue is fixed with the new HA OS 10.3 release. Now internal Zigbee is working nicely together with the Homematic RF PCB on the J11 header. 
Thanks for everybody and especially @jens-maus for the awesome support! 😊

**Comment by bogosavljevic** [comment]
Hi All,

Can anyone share some wiring or pictures how to connect HM-MOD-RPI-PCB with yellow on J11 Header.

Thanks
